bundles/org.eclipse.packagedrone.utils.rpm/src/org/eclipse/packagedrone/utils/rpm/build/RpmBuilder.java

```diff
@@ -139,6 +142,89 @@ public String toString ()
         }
     }
 
+    public static class Feature
+    {
+        private String name;
+
+        private String evr;
```

Comment:
I don't really know what `evr` means, maybe `version` is a better name.

Reply:
The evr means epoch-version-release. I think you call it version elsewhere, but that is somewhat ambiguous, since a version is really one part of a three-tuple.

Reply:
Ok, then I would suggest to go for `epochVersionRelease` … Java is verbose after all :wink:

Reply:
OK, unless we just extend `Dependency`, then it would just inherit `getVersion()` from that. It looks like a feature is just a dependency with a description. I didn't actually see where this description is actually used in the rpm code.